src/shared/components/login/login.js

```diff
@@ -56,15 +56,20 @@ class Login extends Component {
   }
 
   render() {
-    const { error } = this.state;
+    let { error } = this.state;
+    if (error === 'Request failed with status code 401') {
```

Comment:
In order to make server response handling more dynamic, let's just say:

`if (error.includes('401')) {`

We know that the server response will almost always at least contain the response code..